### DISCUSSION TITLE
How to configure assets at system level?

### DISCUSSION THREAD

**Discussion by pranavirabatti** [Sunbird-Knowlg/Community#39] [General Discussions]
**Sunbird portal version 4.4.1**

Is there any way to pass 'all' or '*' keyword just like we configure form at the system level? Or do I have to create assets for every organization we create?

```
curl --location --request POST '{{host}}/api/asset/v1/create' \
--header 'Content-Type: application/json' \
--header 'Authorization: {{api_key}}' \
--header 'x-authenticated-user-token: {{auth_user_token}}' \
--header 'X-Channel-ID: {{orgId}}' \
--data-raw '{
    "request": {
        "asset": {
            "name": "SVG_TEMPLATE_LAYOUT_NAME",
            "identifier": "SVG_TEMPLATE_LAYOUT_ID",
            "code": "SVG_TEMPLATE_LAYOUT_ID",
            "mimeType": "image/svg+xml",
            "license": "CC BY 4.0",
            "primaryCategory": "Certificate Template",
            "mediaType": "image",
            "certType": "cert template layout",
	 "data": {
                "title": "SVG_TEMPLATE_LAYOUT_NAME"
            },
            "issuer": {
                "name": "",
                "url": ""
            },
            "signatoryList": []
        }
    }
}'
```

@vinukumar-vs can you please help?

**Comment by pranavirabatti** [comment]
Any update on this?

**Reply by AmiableAnil** [reply]
@pranavirabatti By using the read API still we can read the other org asset. Knowlg is not restricting it, but in middleware or OPA layer we can add validation for channel. Also by using the composite search we can access the other org asset until we are not filtering based on channel.

@vinukumar-vs

**Reply by pranavirabatti** [reply]
I thought about it. Thanks!

**Reply by pranavirabatti** [reply]
@AmiableAnil @vinukumar-vs That's the only way for now? 
cc: @komalm

**Reply by AmiableAnil** [reply]
Yes @pranavirabatti

**Reply by pallakartheekreddy** [reply]
@pranavirabatti As we haven't received a response from you, we will proceed to close this thread. If you have any further questions or require assistance in the future, please don't hesitate to start a new thread. Thank you.
